.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["env", "react"]
```

Comment:
Presets for babel. These can be included in webpack.config.js but is best to keep external configurations in external file.